src/Page.php

```diff
@@ -99,8 +99,8 @@ private static function format(string $content): string
         return $content;
     }
 
-    private static function getCacheDir(): string
+    private static function getCacheDir(string $subDir = ''): string
     {
-        return "{$_SERVER['HOME']}/.tldr";
+        return "{$_SERVER['HOME']}/.tldr".$subDir;
```

Comment:
I think we can just hardcode it here. so it'll be `"{$_SERVER['HOME']}/.tldr/pages"`

Reply:
Yeh, fair enough. 😋